flink-table/flink-table-planner-blink/src/test/resources/org/apache/flink/table/planner/plan/stream/sql/DagOptimizationTest.xml

```diff
@@ -212,7 +211,8 @@ Sink(name=[`default_catalog`.`default_database`.`appendSink1`], fields=[a, b1],
 
 Sink(name=[`default_catalog`.`default_database`.`appendSink2`], fields=[a, b1], updateAsRetraction=[false], accMode=[Acc])
 +- Union(all=[true], union=[a1, b1], updateAsRetraction=[false], accMode=[Acc])
-   :- Reused(reference_id=[1])
+   :- Calc(select=[a AS a1, b AS b1], updateAsRetraction=[false], accMode=[Acc])
```

Comment:
It is right that updateAsRetraction is false for this Calc, but the updateAsRetraction is not right for its input Calc which should also be false.